edx_rest_api_client/auth.py

```diff
@@ -17,18 +17,19 @@ def __init__(self, username, full_name, email, signing_key, issuer=None, expires
         self.tracking_context = tracking_context
 
     def __call__(self, r):
-
+        now = datetime.datetime.utcnow()
         data = {
             'username': self.username,
             'full_name': self.full_name,
             'email': self.email,
+            'iat': now,
```

Comment:
Yep. That's a bug that has been here since this was added. Please file a bug ticket to fix it. I can't recall ever using a value of 0, so it's not high priority.